src/containers/guest-home-page/faq/Faq.tsx

```diff
@@ -51,26 +51,16 @@ const Faq = () => {
         <Box
           key={item.question} ///question as key
```

Comment:
Is this comment necessary?

Reply:
not necessary

Reply:
this comment has to be removed because the code is self-explanatory